components/common/UserPicture.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
No, it's not safe, would error `React is not defined`.
Because JSX wants React to be in scope.